### ISSUE TITLE
Armor rating is missing

### ISSUE THREAD

**Issue by Odie** [Odie/skyui-vr#21]
As reported here:
https://www.reddit.com/r/skyrimvr/comments/9tdms2/sky_ui/

**Comment by Skidrow68** [comment]
Yes anoying that it dont show on armour and weapon if it is better or not then the one you have on you like it do without the mod.

**Comment by Skidrow68** [comment]
Sry dont realy get what you ask me to do?

**Comment by baggyg** [comment]
You can see the responsible code in:

 https://github.com/Odie/skyui-vr/blob/master/src/ItemMenus/BottomBar.as

on line 86 I believe. If you visit a shop or alternatively are looting a dead body and see a piece of armour in the bottom bar it should give you the armour rating and in brackets the difference to the similar piece of armour you are wearing. I.e. find a helmet that has 35 armour rating. You are wearing a helmet with rating 30 therefore it should say "35 (+5)". Instead that area is always blank for some reason.

**Comment by Odie** [comment]
Hi all,

Looked into this. While investigating this, I found that rebuilding bottombar.fla fixed the issue. It's a little disturbing this sort of thing can happen since compiling these files are more or less automated. In any case... the newly rebuilt version is available here:

https://github.com/Odie/skyui-vr/releases/download/v1.0-beta.3/SkyUI.-.VR.v1.0-beta.3.rebuilt.7z

The version was not incremented because no fixes or features were added. Files were simply rebuilt.
